ast/src/utils.rs

```diff
@@ -93,3 +93,24 @@ pub fn get_use_lsp() -> bool {
     }
     false
 }
+
+pub fn create_synthetic_key_from_ref(node_ref: NodeRef, start: u32) -> String {
```

Comment:
you forgot the verb here 

https://github.com/stakwork/stakgraph/blob/main/mcp/src/graph/utils.ts#L76